specification/peering/resource-manager/Microsoft.Peering/stable/2021-01-01/peering.json

```diff
@@ -0,0 +1,3540 @@
+        "parameters": [
+          {
+            "name": "peeringLocation",
+            "in": "query",
+            "description": "The location of the peering.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "kind",
+            "in": "query",
+            "description": "The kind of the peering.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Direct",
+              "Exchange"
+            ]
+          },
+          {
+            "name": "asn",
+            "in": "query",
+            "description": "The ASN number associated with a legacy peering.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PeeringListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List legacy peerings": {
+            "$ref": "./examples/ListLegacyPeerings.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Peering/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available API operations for peering resources.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List peering operations": {
+            "$ref": "./examples/ListPeeringOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/peerAsns/{peerAsnName}": {
+      "get": {
+        "tags": [
+          "PeerAsns"
+        ],
+        "description": "Gets the peer ASN with the specified name under the given subscription.",
+        "operationId": "PeerAsns_Get",
```

Comment:
PeerAsns will become the class name, will people know what an Asn is? or should this be more verbose?